CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 All changes to the project will be documented in this file.
 
 ## [1.37.2] - Not Yet Released
+* Mono versions update in the readme (PR: [#1929](https://github.com/OmniSharp/omnisharp-roslyn/pull/1929))
```

Comment:
we only add changes that are visible for consumers as features/bugs and skip things like readme/documentation updates, build script changes or test infra modifications. could you please get rid of them?

Reply:
Done.